src/main/scripts/prepare_release.py

```diff
@@ -0,0 +1,180 @@
+import git
+import logging as log
+import os
+import re
+import subprocess
+import sys
+from argparse import ArgumentParser
+from datetime import datetime
+from ruamel.yaml import YAML
+from tempfile import mkstemp
+
+products = ["bamboo", "bamboo-agent",
+            "bitbucket", "confluence", "crowd", "jira"]
+prodbase = "src/main/charts"
+
+
+def get_chart_versions():
+    versions = {}
+    for prod in products:
+        proddir = f"{prodbase}/{prod}"
+        chartfile = f"{proddir}/Chart.yaml"
+
+        with open(chartfile, 'r') as chart:
+            yaml = YAML()
+            chartyaml = yaml.load(chart)
+
+        versions[prod] = {}
+        versions[prod]['kubeVersion'] = chartyaml['kubeVersion']
+        versions[prod]['appVersion'] = chartyaml['appVersion']
+
+        log.info(f"Product {prod} versions: {versions[prod]['kubeVersion']}, {versions[prod]['appVersion']}")
+
+    return versions
+
+
+def gen_changelog(product, path):
+    repo = git.Repo(path)
+    cli = git.Git(path)
+    tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime)
+
+    lasttag = tags[-1]
+    tagver = re.sub(r'^[^-]+-', '', lasttag.name)
+    log.info(f'Generating {product} changelog since {tagver}')
+
+    # git log will pick up commits in src/main/charts/$product directory only
+    changelog = cli.log(f'{lasttag}..main', "--", prodbase + '/' + product, graph=True, pretty='format:%s',
+                        abbrev_commit=True, date='relative', )
+
+    if not changelog:
+        # It is possible that there are no commits to the Helm chart,
+        # in this case we just write a generic git log
+        log.info(f'No commits to {product} Helm chart found')
+        default_git_log = '* Update Helm chart version'
```

Comment:
We can change this message, but in any case having empty release notes isn't good